src/app/page.tsx

```diff
@@ -1,16 +1,66 @@
+'use client'
+import {
+  CodeMarkdown,
+  Container,
+  FormGithub,
+  MarkdownPreview,
+  NavBar,
+} from '@/components'
+import { useEffect, useState } from 'react'
+
 export default function Home() {
+  const [markdown, setMarkdown] = useState('')
+  const [showCodeMarkdown, setshowCodeMarkdown] = useState(false)
+  const [isLoading, setIsLoading] = useState(false)
+
+  useEffect(() => {
+    const githubProfile =
+      '<div style="display: flex; justify-content: center; align-items: center; height: 100%;">\n\n' +
+      '# BIENVENIDO A PERFIGIT\n\n' +
+      '</div>'
+
+    setMarkdown(githubProfile)
+  }, [])
+  const toggleMarkdownPreview = () => {
+    setshowCodeMarkdown(!showCodeMarkdown)
+  }
+  const handleSubmit = (value: string) => {
+    setIsLoading(true)
+    console.log(value)
+    // setIsLoading(false)
+  }
+
   return (
-    <main className=''>
-      <div className='navbar bg-base-100'>
-        <div className='flex-1'>
-          <a className='btn btn-ghost text-xl'>PerfiGit</a>
-        </div>
-        <div className='flex-none'>
-          <a className='btn btn-outline' href=''>
-            Explore
-          </a>
-        </div>
-      </div>
+    <main>
+      <header>
+        <Container className='mt-8 flex h-20 items-center'>
+          <NavBar />
+        </Container>
+      </header>
+      <section>
+        <Container className='mt-8'>
+          <div className='p card h-[780px] overflow-hidden bg-neutral shadow-xl lg:card-side'>
+            {showCodeMarkdown ? (
+              <CodeMarkdown
+                code={markdown}
+                onCodeChange={setMarkdown}
+                onClick={toggleMarkdownPreview}
+              />
+            ) : (
+              <FormGithub
```

Comment:
Cambiar el nombre del componente a `GIthubForm`
```suggestion
              <GithubForm
```